pkg/kubelet/status/status_manager.go

```diff
@@ -682,34 +682,47 @@ type SidecarsStatus struct {
 }
 
 // GetSidecarsStatus returns the SidecarsStatus for the given pod
-func GetSidecarsStatus(pod *v1.Pod) SidecarsStatus {
+func GetSidecarsStatus(pod *v1.Pod, caller string) SidecarsStatus {
```

Comment:
Passing args just for logs seems wrong to me outside a need for debug.
I don't think we should keep those long term.

Reply:
Agreed, it's really just for debug. Though honestly given these calls interleave having some kind of trace id is almost vital. I'm in 2 minds about merging the PR with it or not.

Reply:
I've removed the caller arg.